sound/soc/intel/boards/sof_sdw_common.h

```diff
@@ -55,6 +55,8 @@ enum {
 #define SOF_SDW_NO_AGGREGATION		BIT(14)
 /* If a CODEC has an optional speaker output, this quirk will enable it */
 #define SOF_CODEC_SPKR			BIT(15)
+/* If the CODEC has additional devices attached directly to it */
+#define SOF_SIDECAR_AMPS		BIT(16)
```

Comment:
is there any dependency between SOF_SIDECAR_AMPS and SOF_CODEC_SPKR ?

Reply:
You should probably add a comment saying that all combinations of (1,0), (0, 1), and (1,1) refer to different speaker layouts.

Reply:
Can do.